.firebaserc

```diff
@@ -1,14 +1,14 @@
 {
   "projects": {
-    "default": "ignite-hq"
+    "default": "ignite"
   },
   "targets": {
-    "ignite-hq": {
+    "ignite": {
       "hosting": {
         "docs.ignite.com": [
           "ignite-go-docs"
         ]
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
@ilgooz should we keep using "ignite-hq" for firebase ?

Reply:
Yes.